src/app/classes/task.ts

```diff
@@ -0,0 +1,16 @@
+import { ITask } from '../../models/task';
+
+export class Task implements ITask {
+  status?: boolean;
+  name?: string;
+  id: string;
+  categoryID: string;
+  order: number;
+  constructor() {
+    this.status = false;
+    this.name = '';
+    this.id = '0';
+    this.categoryID = '0';
+    this.order = 0;
+  }
+}
```

Comment:
Conventionally, we don't use classes when defining data types for Angular apps.
Use interfaces instead and manage initial values in components, services, etc. that are using them.